tests/policies/test_policies.py

```diff
@@ -89,18 +80,68 @@ def test_serialize_identity(env_name, model_cfg, normalize, tmpdir):
         rng=np.random.RandomState(0),
     )
 
-    serialize.save_stable_model(tmpdir, model, vec_normalize)
-    # We use `orig_venv` since `load_policy` automatically wraps `loaded`
-    # with a VecNormalize, when appropriate.
-    loaded = serialize.load_policy(model_name, tmpdir, orig_venv)
-    orig_venv.env_method("seed", 0)
-    orig_venv.reset()
+    serialize.save_stable_model(tmpdir, model)
+    loaded = serialize.load_policy(model_name, tmpdir, venv)
+    venv.env_method("seed", 0)
+    venv.reset()
     new_rollout = rollout.generate_transitions(
         loaded,
-        orig_venv,
+        venv,
         n_timesteps=1000,
         deterministic_policy=True,
         rng=np.random.RandomState(0),
     )
 
     assert np.allclose(orig_rollout.acts, new_rollout.acts)
+
+
+class ZeroModule(nn.Module):
+    """Module that always returns zeros of same shape as input."""
+
+    def __init__(self, features_dim: int):
+        """Builds ZeroModule."""
+        super().__init__()
+        self.features_dim = features_dim
+
+    def forward(self, x: th.Tensor) -> th.Tensor:
+        """Returns zeros of same shape as `x`."""
+        assert x.shape[1:] == (self.features_dim,)
+        return x * 0
```

Comment:
Extremely minor nit: this will produce NaNs if `x` has NaNs or infs. `th.zeros_like` will not fail. IDK which is preferable for testing purposes.